### PR TITLE
[EventHubs]: Migrating to Autorest Powershell

### PR DESCRIPTION
**Pull request by damodaravadhani**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/EventHub/11 [hunk 1]
```diff
@@ -0,0 +1 @@
+10
```
Comment (wyunchi-ms) [inline src/EventHub/11:1]: Please remove this.

File: src/EventHub/13 [hunk 2]
```diff
@@ -0,0 +1 @@
+12
```
Comment (wyunchi-ms) [inline src/EventHub/13:1]: Please remove this.

File: src/EventHub/EventHub.Autorest/examples/Approve-AzEventHubPrivateEndpointConnection.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Description                  :
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.EventHub/namespaces/{namespace}/privateEndpointC
+                               onnections/00000000000
+Location                     : Australia East
+Name                         : 00000000000
+PrivateEndpointId            : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}
+ProvisioningState            : Succeeded
+ResourceGroupName            : {resourceGroup}
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Approve-AzEventHubPrivateEndpointConnection.md:17]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Approve-AzEventHubPrivateEndpointConnection.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ConnectionState              : Approved
+Description                  :
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.EventHub/namespaces/{namespace}/privateEndpointC
+                               onnections/00000000000
+Location                     : Australia East
+Name                         : 00000000000
+PrivateEndpointId            : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}
+ProvisioningState            : Succeeded
+ResourceGroupName            : {resourceGroup}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Approve-AzEventHubPrivateEndpointConnection.md:34]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Deny-AzEventHubPrivateEndpointConnection.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+Description                  :
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.EventHub/namespaces/{namespace}/privateEndpointC
+                               onnections/00000000000
+Location                     : Australia East
+Name                         : 00000000000
+PrivateEndpointId            : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}
+ProvisioningState            : Succeeded
+ResourceGroupName            : {resourceGroup}
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Deny-AzEventHubPrivateEndpointConnection.md:17]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Deny-AzEventHubPrivateEndpointConnection.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Deny-AzEventHubPrivateEndpointConnection.md:36]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHub.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+MessageRetentionInDays       : 1
+Name                         : {eventhub}
+PartitionCount               : 1
+PartitionId                  : {0}
+ResourceGroupName            : {resourceGroup}
+SizeLimitInBytes             :
+SkipEmptyArchive             :
+Status                       : Active
+StorageAccountResourceId     :
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHub.md:28]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHub.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+### Example 2: List All EventHubs in a namespace
+```powershell
+Get-AzEventHub -ResourceGroupName {resourceGroup} -NamespaceName {namespace
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHub.md:33]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubApplicationGroup.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+                                 "metricId": "OutgoingMessages"
+                               }, {
+                                 "name": "throttlingPolicy2",
+                                 "type": "ThrottlingPolicy",
+                                 "rateLimitThreshold": 11111,
+                                 "metricId": "OutgoingBytes"
+                               }}
+ResourceGroupName            : myResourceGroup
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubApplicationGroup.md:26]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubApplicationGroup.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+### Example 2: Lists all application groups in an EventHub namespace
+```powershell
+Get-AzEventHubApplicationGroup -ResourceGroupName myResourceGroup -NamespaceName myNamespace
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubApplicationGroup.md:30]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubAuthorizationRule.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+```output
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/authorizationRules
+                               /myAuthRule
+Location                     : Central US
+Name                         : myAuthRule
+ResourceGroupName            : myResourceGroup
+Rights                       : {Listen, Manage, Send}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubAuthorizationRule.md:13]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubAuthorizationRule.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+```output
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/eventhubs/myEventHub/authorizationRules
+                               /myAuthRule
+Location                     : Central US
+Name                         : myAuthRule
+ResourceGroupName            : myResourceGroup
+Rights                       : {Listen, Manage, Send}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubAuthorizationRule.md:27]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubCluster.md [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/clusters/myCluster
+Location                     : australiaeast
+MetricId                     : PROD-00-000
+Name                         : TestClusterAutomatic
+ResourceGroupName            : AutomatedPowershellTesting
+SkuName                      : Dedicated
+Status                       :
+SupportsScaling              : False
+Tag                          : {}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubCluster.md:18]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubClusterNamespace.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+```powershell
+Get-AzEventHubClusterNamespace -ResourceGroupName myResourceGroup -ClusterName myCluster
+```
+
+```output
+Id
+--
+/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace1
+/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace2
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubClusterNamespace.md:11]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubClustersAvailableRegion.md [hunk 15]
```diff
@@ -0,0 +1,4 @@
+### Example 1: Get all available regions for cluster creation
+```powershell
+PS C:\> Get-AzEventHubClustersAvailableRegion
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubClustersAvailableRegion.md:4]: Please add description for each example. And remove the prompt

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubConsumerGroup.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+```output
+CreatedAt                    : 9/13/2022 9:20:47 AM
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace
+                               /eventhubs/eh1/consumergroups/$Default
+Location                     : australiaeast
+Name                         : $Default
+ResourceGroupName            : myResourceGroup
+UpdatedAt                    : 9/13/2022 9:20:47 AM
+UserMetadata                 :
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubConsumerGroup.md:15]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubConsumerGroup.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+
+### Example 2: List all consumer groups in an EventHub entity
+```powershell
+Get-AzEventHubConsumerGroup -NamespaceName myNamespace -ResourceGroupName myResourceGroup -EventHubName myEventHub
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubConsumerGroup.md:20]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubGeoDRConfiguration.md [hunk 18]
```diff
@@ -0,0 +1,17 @@
...
+AlternateName                     :
+Id                                : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/disasterRecoveryConfigs/myAlias
+Location                          :
+Name                              : myalias
+PartnerNamespace                  : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/mySecondaryNamespace
+PendingReplicationOperationsCount : 0
+ProvisioningState                 : Succeeded
+ResourceGroupName                 : myResourceGroup
+Role                              : Primary
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubGeoDRConfiguration.md:16]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubKey.md [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+
+```output
+AliasPrimaryConnectionString   :
+AliasSecondaryConnectionString :
+KeyName                        : RootManageSharedAccessKey
+PrimaryConnectionString        : 000000000000
+PrimaryKey                     : 000000000000
+SecondaryConnectionString      : {ConnectionString}
+SecondaryKey                   : {ConnectionString}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubKey.md:14]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubKey.md [hunk 20]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubKey.md:29]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubNetworkRuleSet.md [hunk 21]
```diff
@@ -0,0 +1,15 @@
...
+```output
+DefaultAction                : Allow
+IPRule                       : {}
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/networkRuleSets/default
+Location                     : Central US
+Name                         : default
+PublicNetworkAccess          : Enabled
+ResourceGroupName            : myResourceGroup
+VirtualNetworkRule           :
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubNetworkRuleSet.md:15]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateEndpointConnection.md [hunk 22]
```diff
@@ -0,0 +1,22 @@
...
+ConnectionState              : Approved
+Description                  :
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.EventHub/namespaces/{namespace}/privateEndpointC
+                               onnections/00000000000
+Location                     : Australia East
+Name                         : 00000000000
+PrivateEndpointId            : /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}
+ProvisioningState            : Succeeded
+ResourceGroupName            : {resourceGroup}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateEndpointConnection.md:16]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateEndpointConnection.md [hunk 23]
```diff
@@ -0,0 +1,22 @@
...
+
+### Example 2: List all private endpoint connections on an EventHub namespace
+```powershell
+Get-AzEventHubPrivateEndpointConnection -ResourceGroupName myResourceGroup -NamespaceName myNamespace
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateEndpointConnection.md:22]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateLink.md [hunk 24]
```diff
@@ -0,0 +1,11 @@
...
+```powershell
+PS C:\>  Get-AzEventHubPrivateLink -ResourceGroupName myResourceGroup -NamespaceName myNamespace
+
+GroupId          : namespace
+Id               : subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/privateLinkResources/namespace
+Name             : namespace
+RequiredMember   : {namespace}
+RequiredZoneName : {privatelink.servicebus.windows.net}
+Type             : Microsoft.EventHub/namespaces/privateLinkResources
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubPrivateLink.md:11]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubSchemaGroup.md [hunk 25]
```diff
@@ -0,0 +1,23 @@
...
+ETag                         : {etag}
+GroupProperty                : {
+                               }
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/schemagroups/mySchemaGroup
+Location                     : Central US
+Name                         : mySchemaGroup
+ResourceGroupName            : myResourceGroup
+SchemaCompatibility          : None
+SchemaType                   : Avro
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubSchemaGroup.md:17]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Get-AzEventHubSchemaGroup.md [hunk 26]
```diff
@@ -0,0 +1,23 @@
...
+
+### Example 2: List all schema groups in an EventHub namespace
+```powershell
+Get-AzEventHubSchemaGroup -ResourceGroupName myResourceGroup -NamespaceName myNamespace
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Get-AzEventHubSchemaGroup.md:23]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHub.md [hunk 27]
```diff
@@ -0,0 +1,56 @@
...
+Location                     : centralus
+MessageRetentionInDays       : 6
+Name                         : myFirstEventHub
+PartitionCount               : 5
+PartitionId                  : {0}
+ResourceGroupName            : myResourceGroup
+SizeLimitInBytes             :
+SkipEmptyArchive             :
+Status                       : Active
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHub.md:27]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHub.md [hunk 28]
```diff
@@ -0,0 +1,56 @@
...
+MessageRetentionInDays       : 6
+Name                         : myFirstEventHub
+PartitionCount               : 5
+PartitionId                  : {0}
+ResourceGroupName            : myResourceGroup
+SizeLimitInBytes             : 11000000
+SkipEmptyArchive             : true
+Status                       : Active
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHub.md:56]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubApplicationGroup.md [hunk 29]
```diff
@@ -0,0 +1,27 @@
...
+                                 "rateLimitThreshold": 10000,
+                                 "metricId": "IncomingMessages"
+                               }, {
+                                 "name": "t2",
+                                 "type": "ThrottlingPolicy",
+                                 "rateLimitThreshold": 20000,
+                                 "metricId": "OutgoingBytes"
+                               }}
+ResourceGroupName            : myResourceGroup
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubApplicationGroup.md:27]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubAuthorizationRule.md [hunk 30]
```diff
@@ -0,0 +1,26 @@
...
+New-AzEventHubAuthorizationRule -ResourceGroupName myResourceGroup -NamespaceName myNamespace -Name myAuthRule -Rights @('Manage', 'Send', 'Listen')
+```
+
+```output
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/authorizationRules/myAuthRule
+Location                     : Central US
+Name                         : myAuthRule
+ResourceGroupName            : myResourceGroup
+Rights                       : {Listen, Manage, Send}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubAuthorizationRule.md:12]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubAuthorizationRule.md [hunk 31]
```diff
@@ -0,0 +1,26 @@
...
+
+```output
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/eventhubs/myEventHub/authorizationRules/myAuthRule
+Location                     : Central US
+Name                         : myAuthRule
+ResourceGroupName            : myResourceGroup
+Rights                       : {Listen, Manage, Send}
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubAuthorizationRule.md:26]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubCluster.md [hunk 32]
```diff
@@ -0,0 +1,38 @@
...
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/clusters/myCluster
+Location                     : australiaeast
+MetricId                     : PROD-00-000
+Name                         : myCluster
+ResourceGroupName            : myResourceGroup
+SkuName                      : Dedicated
+Status                       :
+SupportsScaling              : True
+Tag                          : {}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubCluster.md:18]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubCluster.md [hunk 33]
```diff
@@ -0,0 +1,38 @@
...
+Location                     : australiaeast
+MetricId                     : PROD-00-000
+Name                         : myCluster
+ResourceGroupName            : myResourceGroup
+SkuName                      : Dedicated
+Status                       :
+SupportsScaling              :
+Tag                          : {}
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubCluster.md:38]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubConsumerGroup.md [hunk 34]
```diff
@@ -0,0 +1,15 @@
...
+
+```output
+CreatedAt                    : 9/13/2022 9:20:47 AM
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace
+                               /eventhubs/eh1/consumergroups/myConsumerGroup
+Location                     : australiaeast
+Name                         : myConsumerGroup
+ResourceGroupName            : myResourceGroup
+UpdatedAt                    : 9/13/2022 9:20:47 AM
+UserMetadata                 : Test ConsumerGroup
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubConsumerGroup.md:14]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubIPRuleConfig.md [hunk 35]
```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
+```powershell
+{{ Add code here }}
+```
+
+```output
+{{ Add output here }}
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubIPRuleConfig.md:7]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubSchemaGroup.md [hunk 36]
```diff
@@ -0,0 +1,17 @@
...
+ETag                         : {etag}
+GroupProperty                : {
+                               }
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/schemagroups/mySchemaGroup
+Location                     : Central US
+Name                         : mySchemaGroup
+ResourceGroupName            : myResourceGroup
+SchemaCompatibility          : Backward
+SchemaType                   : Avro
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubSchemaGroup.md:17]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubThrottlingPolicyConfig.md [hunk 37]
```diff
@@ -0,0 +1,22 @@
...
+{{ Add code here }}
+```
+
+```output
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
+### Example 2: {{ Add title here }}
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubThrottlingPolicyConfig.md:12]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/New-AzEventHubVirtualNetworkRuleConfig.md [hunk 38]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+```output
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
+### Example 2: {{ Add title here }}
+```powershell
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/New-AzEventHubVirtualNetworkRuleConfig.md:13]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Set-AzEventHub.md [hunk 39]
```diff
@@ -0,0 +1,59 @@
...
+IntervalInSeconds            : 600
+Location                     : centralus
+MessageRetentionInDays       : 6
+Name                         : myFirstEventHub
+PartitionCount               : 5
+PartitionId                  : {0}
+ResourceGroupName            : myResourceGroup
+SizeLimitInBytes             : 11000000
+SkipEmptyArchive             : true
+Status                       : Active
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Set-AzEventHub.md:26]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Set-AzEventHubAuthorizationRule.md [hunk 40]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+```output
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/authorizationRules/myAuthRule
+Location                     : Central US
+Name                         : myAuthRule
+ResourceGroupName            : myResourceGroup
+Rights                       : {Listen, Manage, Send}
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Set-AzEventHubAuthorizationRule.md:13]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Set-AzEventHubAuthorizationRule.md [hunk 41]
```diff
@@ -0,0 +1,27 @@
...
+Set-AzEventHubAuthorizationRule -ResourceGroupName myResourceGroup -NamespaceName myNamespace -EventHubName myEventHub -Name myAuthRule -Rights @('Manage', 'Send', 'Listen')
+```
+
+```output
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace/eventhubs/myEventHub/authorizationRules/myAuthRule
+Location                     : Central US
+Name                         : myAuthRule
+ResourceGroupName            : myResourceGroup
+Rights                       : {Listen, Manage, Send}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Set-AzEventHubAuthorizationRule.md:25]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Set-AzEventHubConsumerGroup.md [hunk 42]
```diff
@@ -0,0 +1,33 @@
...
+```output
+CreatedAt                    : 9/13/2022 9:20:47 AM
+Id                           : /subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.EventHub/namespaces/myNamespace
+                               /eventhubs/eh1/consumergroups/myConsumerGroup
+Location                     : australiaeast
+Name                         : myConsumerGroup
+ResourceGroupName            : myResourceGroup
+UpdatedAt                    : 9/13/2022 9:20:47 AM
+UserMetadata                 : Example Metadata
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Set-AzEventHubConsumerGroup.md:32]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Set-AzEventHubConsumerGroup.md [hunk 43]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Set-AzEventHubConsumerGroup.md:15]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Set-AzEventHubNetworkRuleSet.md [hunk 44]
```diff
@@ -0,0 +1,88 @@
...
+                                   "id": "/subscriptions/{subscriptionId}/resourcegroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myVirtualNetwork/subnets/default"
+                                 },
+                                 "ignoreMissingVnetServiceEndpoint": false
+                               },{
+                                 "subnet": {
+                                   "id": "/subscriptions/{subscriptionId}/resourcegroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myVirtualNetwork/subnets/mySubnet"
+                                 },
+                                 "ignoreMissingVnetServiceEndpoint": false
+                               }}
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Set-AzEventHubNetworkRuleSet.md:87]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Test-AzEventHubName.md [hunk 45]
```diff
@@ -0,0 +1,23 @@
+### Example 1: Check the availability of an EventHub namespace name
+```powershell
+Test-AzEventHubName -NamespaceName myNamespace
+```
+
+```output
+Message                                                                                                NameAvailable Reason
+-------                                                                                                ------------- ------
+The specified name is not available. For more information visit https://aka.ms/eventhubsarmexceptions.         False NameInUse
+```
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Test-AzEventHubName.md:10]: Please add description for each example.

File: src/EventHub/EventHub.Autorest/examples/Test-AzEventHubName.md [hunk 46]
```diff
@@ -0,0 +1,23 @@
...
+```powershell
+Test-AzEventHubName -NamespaceName myNamespace -ResourceGroupName myResourceGroup -AliasName myAlias
+```
+
+```output
+Message                                                                                                NameAvailable Reason
+-------                                                                                                ------------- ------
+The specified name is not available. For more information visit https://aka.ms/eventhubsarmexceptions.         False NameInUse
+```
+
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub.Autorest/examples/Test-AzEventHubName.md:22]: Please add description for each example.

**Comment by damodaravadhani** [comment]
Hello @wyunchi-ms , I have pushed all fixes discussed in Monday office hours, however a couple cmdlets still fail during Playback saying "Invalid Context" or "ResourceGroup not found" which I was not able to fix. Apart from this, there is nothing left from my side to push. Please feel free to add "-skip" tag to tests that might block you from merging because all pass when run live. Thank you !

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**